Pacemaker/pacemaker_remote_poc/README.md

```diff
@@ -0,0 +1,36 @@
+### Conceptual Overview
+
+- A regular pacemaker cluster with communication using corosync can be extended up to 32 nodes. With the pacemaker_remote service, High Availability clusters can be extended to include additional nodes beyond this limit.
+- The pacemaker_remote service can be operated as a physical node (remote node) or as a virtual node (guest node). Unlike normal cluster nodes, both remote and guest nodes are managed by the cluster as resources.
+- From the resource management point of view, they behave as regular cluster nodes.
+- Remote nodes do not need to have the full cluster stack installed, as they only run the pacemaker_remote service. The service acts as a proxy, allowing the cluster stack on the“regular”cluster nodes to connect to the service.
+
+### Terminology
+###### **Cluster Node**: A node that runs the complete cluster stack.
+A regular cluster node can perform following:
+- Run cluster resources.
+- Run all command line tools, such as crm , crm_mon.
+- Execute fencing actions.
+- Count toward cluster quorum.
+- Serve as the designated coordinator (DC).
+
+###### **Pacemaker Remote (systemd service: pacemaker_remote )**
+- A service daemon that makes it possible to use a node as a Pacemaker node without deploying the full cluster stack.
+
+###### **Remote Node: A physical machine that runs the pacemaker_remote daemon**
+- A special resource( ocf:pacemaker:remote ) needs to run on one of the cluster nodes to manage communication between the cluster node and the remote node
```

Comment:
What is the expected behavior if the remote node monitoring resource gets deleted?

Reply:
You question is not clear actually.
Remote Node is manged using pacemaker_remote service which runs on a remote node. If that service is deleted or stopped, then remote resource management will not happen through pacemaker.
Remote node itself is created as a resource in the cluster. If that resource is deleted. That remote node management and hence services running on that remote node will not be managed by pacemaker.

Reply:
I read somewhere that the remote node is fenced after deleting the resource and the expected result is resource is deleted without fencing of the remote node. 
https://bugzilla.redhat.com/show_bug.cgi?id=1390609

I am not sure the same behavior is still present with the latest pacemaker and is there any impact on cortx?

Reply:
I haven't seen this behavior yet on VM. I tried this multiple times. 
pacemaker RPM they mentioned is: 1.1.15 and my RPM is 1.1.20. Not sure if its fixed in this version.

Reply:
Okay. Then we are good.